packages/admin-ui/src/combobox/combobox.style.ts

```diff
@@ -0,0 +1,75 @@
+import { style } from '@vtex/admin-ui-core'
+
+const height = 48
+const width = '346px'
+const paddingX = 3
+
+export const container = style({
+  width,
+})
+
+export const input = style({
+  width,
+  paddingX,
+  height,
+  text: '$body',
+  paddingTop: 4,
+  bg: '$form.neutral',
+  border: '$form.neutral',
+  borderRadius: 'default',
+  marginY: 1,
+  color: '$form.neutral',
+  outline: 0,
+  transition: 'snap',
+  ':hover': {
+    border: '$form.neutralHover',
+  },
+  ':focus': {
+    border: '$form.neutralFocus',
+    boxShadow: '$ring.neutral',
+  },
+  ':disabled': {
+    bg: '$disabled',
+    color: '$disabled',
+  },
+})
+
+export const buttonContainer = style({
+  right: 0,
+  top: 1,
+  height: 46,
+  paddingRight: 3,
+  position: 'absolute',
+  display: 'flex',
+  color: '$primary',
+})
+
+export const clearButton = style({
+  marginTop: 2,
+  marginRight: 1,
+  color: '$secondary',
+})
+
+export const popover = style({
```

Comment:
I'm not sure if it's behavior from our doc site or from the Combobox popover, but I think it's missing a `zIndex` value here
![Captura de Tela 2022-03-09 às 11 33 40](https://user-images.githubusercontent.com/20579226/157462512-b51cece1-9181-480b-b97a-13219050ab68.png)